### PR TITLE
Remove router synchronizer

### PR DESCRIPTION
**Pull request by tbachman**
The router synchronizer is no longer needed, as there
is a CLI command available to force the sync manually.

Signed-off-by: Thomas Bachman tbachman@yahoo.com

### CODE REVIEW

**Comment by tbachman** [comment]
Updated with latest patch set.

File: apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py [hunk 1]
```diff
@@ -57,6 +57,7 @@
 from apic_ml2.neutron.plugins.ml2.drivers.cisco.apic import nova_client
 from apic_ml2.neutron.plugins.ml2.drivers.cisco.apic import rpc as t_rpc
 
+DEFAULT_APIC_SYNC_INTERVAL = 30
```
Comment (amittbose) [inline apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py]: why is this needed?

**Comment by amittbose** [comment]
Should the commit title change to reflect removing the sync stuff all over?
